python/pyspark/sql/connect/dataframe.py

```diff
@@ -201,6 +202,34 @@ def sort(self, *cols: "ColumnOrString") -> "DataFrame":
         """Sort by a specific column"""
         return DataFrame.withPlan(plan.Sort(self._plan, *cols), session=self._session)
 
+    def sample(
```

Comment:
The pyspark dataframe API has
```
    @overload
    def sample(self, fraction: float, seed: Optional[int] = ...) -> "DataFrame":
        ...

    @overload
    def sample(
        self,
        withReplacement: Optional[bool],
        fraction: float,
        seed: Optional[int] = ...,
    ) -> "DataFrame":
        ...
```

Can we match?

Reply:
I guess we can discard those ones ? @HyukjinKwon

Reply:
Maybe my real question was, will we have an issue to be compatible with existing pyspark dataframe code (needs different imports, of course) if we discard such API? I see many other similar API existing for pyspark dataframe.

Reply:
users may have to change their codes for this emigration, but I think this is also a chance to make some changes.

Reply:
Sure. We also can go to that direction.